docs/FAQ.md

```diff
@@ -0,0 +1,74 @@
+
+<hr>
+
+**Q : Are there any limitations to using fAIr?**
+
+A : While fAIr offers powerful AI models, it's important to note that the accuracy of the results may vary depending on the quality and resolution of the input imagery and the complexity of the mapping task. Some tasks may require additional manual editing or validation. It's recommended to review the documentation and understand the limitations of each model before use.
+
+<hr>
+
+**Q : How can I contribute feedback or report issues with fAIr?**
+
+A : You can contribute to the improvement of fAIr by providing feedback, reporting issues, or suggesting new features through the project's GitHub repository. By creating an issue, you can share your observations or difficulties faced while using fAIr, allowing the developers to address them and enhance the toolkit.
+
+<hr>
+
+**Q : Can I integrate fAIr with other mapping tools or platforms?**
+
+A : Yes, fAIr is designed to be modular and can be integrated into existing mapping workflows and platforms. It provides interoperability with tools like OpenStreetMap, QGIS, and ArcGIS. You can leverage the AI models from fAIr within these tools or use the outputs of fAIr in combination with other mapping datasets and tools.
+
+<hr>
+
+## Developers
+
+**Q : How can I contribute to the development of fAIr?**
+
+A : You can contribute to fAIr's development by following the guidelines provided in the project's GitHub repository. You can contribute code, suggest new features, or help improve documentation. By submitting pull requests, you can propose changes and collaborate with the project maintainers and the community.
+
+<hr>
+
+**Q : What programming languages and frameworks are used in fAIr?**
+
+A : fAIr frontend is made with React and backend with Django.fAIr is primarily developed using Python, which is widely used in machine learning and AI. It leverages popular frameworks such as PyTorch and TensorFlow for model development and training. Familiarity with Python and these frameworks will be beneficial if you plan to work on fAIr's development.
+
+<hr>
+
+**Q : Are there any specific coding standards or guidelines for contributing to fAIr?**
+
+A : Yes, fAIr follows coding standards and guidelines outlined in the project's repository. These guidelines cover aspects such as code formatting, documentation, and testing. Adhering to these standards helps maintain code consistency and facilitates collaboration among contributors.
+
+<hr>
+
+**Q : Can I propose new AI models or enhancements to existing models?**
+
+A : Absolutely! The fAIr project welcomes contributions related to new AI models or enhancements to existing models. If you have developed a new model that aligns with the project's goals or if you have ideas to improve the current models, you can propose them through the GitHub repository and engage in discussions with the community or you can create them at [here](https://fair-dev.hotosm.org/ai-models) .
```

Comment:
Advised to Remove last or part , once approved community will be able go use it , link them to fAIr-utilities

Reply:
Removed the last or part . But I am unable to get the fAIr utilities.

Reply:
@neelimagoogly fAIr repo is the one handing backend and frontend components .. However, the backend is using s separate library we call it fAIr-utilities to handle:

1. Data preparation for the models 
2. models trainings
3. inference process
4. post processing (converting the predicted features to geo data)

Reply:
Thanks @omranlm . I have made the change and linked it to fAIr utilities.